perfkitbenchmarker/benchmarks/netperf_benchmark.py

```diff
@@ -31,11 +31,22 @@
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.packages import netperf
 
+flags.DEFINE_integer('netperf_max_iter', None,
+                     'Maximum number of iterations to run during '
+                     'confidence interval estimation. If unset, '
+                     'a single iteration will be run.',
```

Comment:
I want to distinguish between performing confidence interval estimation and not, which was the reason for using None here.
netperf will silently ceil the max iters to 3 (docs [here](http://www.netperf.org/svn/netperf2/tags/netperf-2.6.0/doc/netperf.html), search for `-i <sizespec>`), so I don't want to blindly pass the value of `--netperf_max_iter`.  But the current lower bound doesn't make sense; I'll update it to a default of `None`, `lower_bound` of 3.